pkg/controller/daemon/daemoncontroller.go

```diff
@@ -539,19 +540,26 @@ func storeDaemonSetStatus(dsClient unversionedextensions.DaemonSetInterface, ds
 		return nil
 	}
 
+	clone, err := conversion.NewCloner().DeepCopy(ds)
```

Comment:
Is this the new convention for invoking deep copy?  I thought we were using the copy method directly now?